### DISCUSSION TITLE
Time entry not working for the 1st day, when the month split happens in-between the week

### DISCUSSION THREAD

**Discussion by rjayaser** [kimai/kimai#2491] [Questions & Answers]
Time entry not working for the 1st day, when the month split happens in-between the week i.e for the last week 1 April 2021 falls in between the week, for that day not able to enter time. please find attached the related screen-shot for reference and let know if any info. on the same.
![month_split_in_week_issue](https://user-images.githubusercontent.com/67286004/113858234-1adb6400-97c1-11eb-8c3d-244e8bae193e.PNG)

**Comment by kevinpapst** [comment]
What? Show me a screenshot fo your lockdown period configuration.

**Reply by rjayaser** [reply]
hi kevin, please find below the screen-shot of the same and let know if any more info. required. Thanks in Advance.

![image](https://user-images.githubusercontent.com/67286004/113864775-eec3e100-97c8-11eb-9b82-5896e1923b90.png)

**Reply by kevinpapst** [reply]
I don't see a problem. Period is locked.

**Reply by rjayaser** [reply]
okay resolved, thanks kevin.
